### DISCUSSION TITLE
Problem with ReactiveUI.Uno.WinUI

### DISCUSSION THREAD

**Discussion by HavenDV** [unoplatform/uno#8115] [General]
Here are two solutions that demonstrate this problem for a WebAssembly application. The uno solution shows a working Reactive.Uno package and expected behavior, the uno_winui solution shows the problem in the standard uno winui dotnet 6 template.
[ReactiveUI.UnoWinUIWhenActivatedIssue.zip](https://github.com/reactiveui/ReactiveUI/files/8068587/ReactiveUI.UnoWinUIWhenActivatedIssue.zip)

Is there a workaround for this problem for a WebAssembly application?

Issue in the ReactiveUI repositories that has been closed: https://github.com/reactiveui/ReactiveUI/issues/3181
PR that has been closed: https://github.com/reactiveui/ReactiveUI/pull/3182

**Comment by HavenDV** [comment]
To solve this problem, I released a package without net6.0-windows TargetFramework:
https://www.nuget.org/packages/H.ReactiveUI.Uno.WinUI

**Comment by hoainho** [comment]
test - please ignore
